docs/source/operations/fabric_verify.md

```diff
@@ -0,0 +1,167 @@
+## DLT Platform 
+
+### Common Components check
+
+#### Common components check ( like flux, namespaces, service accounts, storage classes)
+
+- Check for pods. Get the list of pods 
+        
+        kubectl get pods -A
+- Check for namespaces.
+
+        kubectl get namespace
+- Check for Storage Classes
+
+        kubectl get sc
+- Check for Service Accounts
+
+        kubeclt get sa -A
+        
+- To describe the pod
+
+        kubectl describe <pod name> -n <namespace>
+-To check the logs of a particular container inside the pod.
+
+        kubectl logs -c <container name in pod> <pod name> -n <namespace>
+
+
+**NOTE:** The pods and other components take some time to setup.You can check the components while the retries occours.
+
+#### Platform specific check
+
+##### Fabric ( Specific components check like ca tools, ca server, orderer n peer)
+
+Stage 1.
+Check for flux pods using, kubectl get pods -A
+
+    kubectl edit deployment/flux-{{ network.env.type }}
+Check the branch details here if they are correct or not.
+
+Stage 2.
+Check for namespaces, service accounts and storage classes.
+Check flux logs using 
+
+    kubectl logs -f <flux pod name>
+
+Possible errors, if namespaces, storege classes etc are not generated.
+Value files are not properly generated or not properly pushed.
+- Check whether git branch is right.
+- Check whether the git credentials are right.
+
+Stage 3. 
+Check for ca-server.
+Check if the pod is up and running.
+If not, Check flux-helm operator  logs..
+Possible errors
+- Value files are not generated right. Release failed
+
+Stage 4.
+Check for orderer Pod.
+Check if the pod is up and running.
+If not, Check flux-helm operator logs..
+Possible errors
+- Value files are not generated right. Release failed
+
+Stage 5.
+Check for peers
+Check if the pod is up and running.
+If not, Check flux-helm operator logs..
+Possible errors
+- Value files are not generated right. Release failed
+For further details check the pod logs or describe the helm release files
+
+Stage 6.
+Create channel/ Join channel
```

Comment:
Also mention the error related to invalid orderer tls certificate (which we fetch from network.yaml's orderer section)